cli/src/test/java/com/devonfw/tools/ide/version/VersionRangeTest.java

```diff
@@ -108,10 +108,10 @@ public void testContains() {
 
     checkContainsNot("(11,22)", "10*");
     checkContains("(11,22)", "11*");
-    checkContains("(11,22)", "11.0*");
-    checkContains("(11,22)", "22*");
-    checkContains("(11,22)", "22.*");
```

Comment:
This we wrong assumptions and assertions.
When implementing a test, we should not assert the behaviour of the implementation but think what the correct result should be and assert that.

Reply:
I am already setting this to resolved since it is fixed with this PR.
BTW: I found the author of this anti-pattern [here](https://github.com/devonfw/IDEasy/commit/bacd9ea001b21bcc788b8267a2c5bb6ebe68a25f#diff-d8d87bce3b2e2c262d5a17b746c91778551cfcb706c4de0ae95b755b68fffcca).
I have to blame myself. Shame on me ;)